AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt

```diff
@@ -1998,6 +1998,8 @@ google.ad,google.ae,google.al,google.am,google.as,google.at,google.az,google.ba,
 ! causes some issues: https://github.com/AdguardTeam/AdguardForMac/issues/550
 ~lafenice.io,~kurer-sreda.ru,~chorki.com,~zattoo.com,~zaycev.net,~xltrax.ee,~hitta.se,~mail.ru,~v7player.wostreaming.net,~facebook.com,~instagram.com,~drive.google.com$$link[rel="manifest"]
 !
+mzh.moegirl.org.cn##.download-app:upward(1)
+mzh.moegirl.org.cn###open-in-app-wrapper
```

Comment:
I suggest this rule `mzh.moegirl.org.cn#?#.header > div:not([class]) > button.download-app:upward(1)`

Reply:
I cannot edit your branch, added the rule directly https://github.com/AdguardTeam/AdguardFilters/commit/b85dbf66aa5a443dc5a594b19c04a3abe8d3af47.
Anyway, thanks for your pull request:)

Reply:
Also note, this type of rules has to be written with `#?#` syntax.